lerobot/common/robot_devices/cameras/opencv.py

```diff
@@ -253,6 +252,9 @@ def connect(self):
         else:
             self.camera = cv2.VideoCapture(self.camera_index)
 
+        if not self.camera.isOpened():
+            raise OSError(f"Camera {self.camera_index} could not be opened.")
```

Comment:
I was getting a strange issue where you could create the camera but it did not open. forgot why exactly but this helped debug it.